flytekit/image_spec/default_builder.py

```diff
@@ -98,14 +101,15 @@
 
 FROM $BASE_IMAGE
 
+WORKDIR /
 USER root
 $APT_INSTALL_COMMAND
 RUN --mount=from=micromamba,source=/etc/ssl/certs/ca-certificates.crt,target=/tmp/ca-certificates.crt \
     [ -f /etc/ssl/certs/ca-certificates.crt ] || \
     mkdir -p /etc/ssl/certs/ && cp /tmp/ca-certificates.crt /etc/ssl/certs/ca-certificates.crt
 
 RUN id -u flytekit || useradd --create-home --shell /bin/bash flytekit
-RUN chown -R flytekit /root && chown -R flytekit /home
+RUN chown flytekit /root && chown -R flytekit /home
```

Comment:
Thanks for your contribution!
Could you elaborate a bit on why is this needed?

Reply:
When running [flyte interactive](https://www.union.ai/docs/flyte/integrations/external-service-backend-plugins/flyteinteractive-plugin/vscode/) where the runtime user is `flytekit` it's convenient to have `/root` be writable by the `flytekit` user.